File: question6.java
import java.util.Scanner;
public class question6 {
    public static void main(String[] args) {
        // girilen sayinin faktoriyelini hesaplayan kod

        Scanner scanner = new Scanner (System.in);

        System.out.println("Faktoriyeli hesaplanacak kod:");

        int sayi=scanner.nextInt();

        int faktoriyel=1;

        for(int i=1;i<=sayi;i++){
            faktoriyel*=i;
        }

        System.out.println("Faktoriyelin sonucu:" + faktoriyel);

        scanner.close();

    }
}

File: question6.py
# girilen sayinin faktoriyelini hesaplayan kod

sayi=int(input("Faktoriyeli hesaplanacak sayi: "))

faktoriyel=1

for i in range(1,sayi+1):
    faktoriyel*=i
    
print("Faktoriyelin sonucu : " + str(faktoriyel))